packages/sandbox/package.json

```diff
@@ -24,5 +25,8 @@
   },
   "volta": {
     "extends": "../../package.json"
+  },
+  "dependencies": {
+    "@dotcom-tool-kit/node": "file:../node"
```

Comment:
**issue:** I think we need `dotcom-tool-kit` installed as a dependency now that we're using the `run:local` hook at runtime

Reply:
why isn't the `node` package a dev dependency then? 😛

Reply:
be...cause... i forgot to add `--save-dev` when i did this